src/frontend/src/utils/requestUtil.ts

```diff
@@ -0,0 +1,23 @@
+import { BASE_URL } from '../constant/api';
+
+export const client = async (
+  endpoint: string,
+  method = 'POST',
+  body = {},
+  headers = {},
+  customConfig = {},
+) => {
+  const options = {
+    method: method,
+    headers: { 'Content-Type': 'application/json', ...headers }, // by default setting the content-type to be json type
+    body: body ? JSON.stringify(body) : null,
+    ...customConfig,
+  };
+
+  const response = await fetch(`${BASE_URL}/${endpoint}`, options);
+  if (!response.ok) {
```

Comment:
Does `response.ok` accepts 201 as well? 🤔 

cz our submission endpoint returns 201 and not 200 ok.

Reply:
I run the test in my local, it works

Reply:
The ok read-only property of the [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) interface contains a Boolean stating whether the response was successful (status in the range 200-299) or not.